python/circuitsvis/attribution.py

```diff
@@ -0,0 +1,181 @@
+from jaxtyping import Float
+from typing import Optional, Union, List, Literal, Tuple
+import torch as t

```

Comment:
all the .py files here use the torch.x convention rather than t.x, would prefer to keep things consistent for now and think about changing the convention everywhere in a follow up PR